tests/test_listener.py

```diff
@@ -0,0 +1,39 @@
+from six.moves import mock
+
+from robotframework_reportportal.listener import listener
+
+ROBOT_SERVICE = 'robotframework_reportportal.service.ReportPortalService'
+
+
+class TestListener:
+
+    @mock.patch(ROBOT_SERVICE)
+    def test_critical_test_failure(self, mock_init, mock_variables,
+                                   suite_attributes, test_attributes):
+        mock_client = mock_init.return_value
+        listener_obj = listener()
```

Comment:
Can we move this line and the next line to a fxiture?

Reply:
Done